packages/node_modules/@ciscospark/internal-plugin-search/src/search.js

```diff
@@ -44,10 +44,6 @@ const Search = SparkPlugin.extend({
     /* eslint max-nested-callbacks: [0] */
     options = options || {};
 
-    if (!options.query) {
-      return Promise.resolve([]);
-    }
```

Comment:
What's the expected use case here @phatv? Is that the only way the files search works with a given convo? Would it be better to say `!options.query || <other param`?

Reply:
Searching for all items. Query is the search string and I don't want to specify that because we want all files. The other params I'm using are sharedIn, type, and limit. None of these are required by all searches and can be left out so checking for another specific param doesn't seem useful.

Reply:
I could check if the object is not empty so we know that it is searching for something. Like `Object.keys(test).length === 0`?

Reply:
I meant will this break any existing search functionality in the client (not your new code)